src/css/cards.less

```diff
@@ -1,3 +0,0 @@
-.__mobiledoc-card {
-  display: inline-block;
```

Comment:
@zeppelin Are you asking how to make cards so they don't appear as `inline-block`?